src/test/java/com/hubspot/jinjava/interpret/JinjavaInterpreterTest.java

```diff
@@ -220,4 +220,24 @@ public void itLimitsOutputSizeWhenSumOfNodeSizesExceedsMax() {
     assertThat(renderResult.getErrors().get(0).getMessage())
       .contains("OutputTooBigException");
   }
+
+  @Test
+  public void itCanPreserveRawTags() {
+    JinjavaConfig preserveRawTagsConfig = JinjavaConfig
+      .newBuilder()
+      .withPreserveRawTags(true)
+      .build();
+    String input = "1{% raw %}2{% endraw %}3";
```

Comment:
Can you also add a test for deferred values? I thought deferred values will not get evaluated normally. Do they get evaluated inside `raw` tags?

Reply:
My testing shows that deferred inside raw does not get evaluated either. https://github.com/HubSpot/jinjava/compare/test-deferred?expand=1

Reply:
Nothing, including deferred tags, will get evaluated inside `raw` tags. However without the `preserveRawTags` flag, on the second pass, the deferred values will get evaluated once the real value is put in the context.

Reply:
In my test above, the deferred values do not get evaluated in the second rendering. Maybe we are not talking about the same case?

Reply:
After line 83:
```
interpreter.getContext().put("deferred", "resolved value");
```
In the second pass, deferred values will no longer be deferred (such as a `contact`, which have the actual contact value in the second pass, rather than being a deferred value). If you run the test after putting that line in, I believe that it will get resolved, which is not what is desired.

Reply:
I think what Jack meant by second pass is another renderer pass without having `DeferredValue.instance()` in the context. But in your test case, the test setUp has `localContext.put("deferred", DeferredValue.instance());` so it gets deferred in the second pass as well. Yo need to remove that from the context to actually resolve them in the second pass.

Reply:
Hmm, what is the use case for those? First round rendering makes it deferred, then second round makes it not deferred, but still don't want to render it.

Reply:
```
<div>
Hi {{ contact.firstname }}!
I personalized this with Jinjava:
{% raw %}Hi {{ contact.firstname }}!{% endraw %}
</div>
```
This should display for someone like:
```
<div>
Hi Jack!
I personalized this with Jinjava:
Hi {{ contact.firstname }}!
</div>
```

Reply:
We wouldn't want it to render like:
```
<div>
Hi Jack!
I personalized this with Jinjava:
Hi Jack!
</div>

Reply:
Got it. Thanks.